src/it/scala/io/iohk/ethereum/sync/FastSyncItSpec.scala

```diff
@@ -234,7 +234,7 @@ class FastSyncItSpec extends FlatSpecBase with Matchers with BeforeAndAfterAll {
     )
   }
 
-  it should "blacklist peer on Invalid batch last header number" in customTestCaseResourceM(
+  ignore should "blacklist peer on Invalid batch last header number" in customTestCaseResourceM(
```

Comment:
why ignoring these two? if the functionality is removed the tests can be removed also, otherwise it's gonna be confusing if someone ever picks the client again to work on it